tests/unit/test_query.py

```diff
@@ -120,7 +128,17 @@ def skin_myo():
     return MappableConcept(
         conceptType="Disease",
         id="normalize.disease.ncit:C167370",
+        primaryCode=code(root="ncit:C167370"),
         label="Skin Myoepithelioma",
+        mappings=[
+            {
+                "coding": {
+                    "code": "C167370",
+                    "system": "https://www.ebi.ac.uk/ols4/ontologies/ncit/classes?short_form=NCIT_",
+                },
+                "relation": "exactMatch",
+            },
+        ],
```

Comment:
obviously will need to remember to change this if the VA-spec discussion about mappableconcepts goes somewhere different